queries/data-transformations/data_transformations/cli.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import os
+import sys
+from typing import Optional
+
+
+def add_common_args(
+    *,
+    parser: argparse.ArgumentParser,
+    batch_size: int = os.cpu_count(),
+    extract_workers: int = 1,
+    transform_workers: int = 1,
+    cleanup_workers: int = 1,
+    extract_queue_size: int = 0,
+    transform_queue_size: int = 0,
+    cleanup_queue_size: int = 0,
+    max_file_processes: int = os.cpu_count(),
+    n_batches: Optional[int] = None,
+):
+    parser.add_argument(
```

Comment:
this is very minor, but it might be good to include the default in these help messages. (e.g. since it is `batch_size: int = os.cpu_count()` above in the function signature you could say "(default: nbr of available CPUs)"

Reply:
That's a good point, yeah I'll do that.